samples/sampler/stories/qa/ContextualPopupDecorator.js

```diff
@@ -44,6 +45,15 @@ const renderSuperTallPopup = () => (
 	</div>
 );
 
+const renderSliderPopup = () => (
+	<IncrementSlider
+		style={{width: '500px'}}
+		max={100}
+		min={0}
+		orientation="horizontal"
```

Comment:
```suggestion
```

It looks same with default value, not needed.

Reply:
It's removed